.github/workflows/build.yml

```diff
@@ -47,6 +47,10 @@ jobs:
           # DNF core plugins are installed in the official CS9 container image
           dnf install -y dnf-plugins-core
 
+          # use legacy crypto policy on el9
+          dnf upgrade -y crypto-policies-scripts crypto-policies
+          update-crypto-policies --set LEGACY
+
```

Comment:
Wouldn't it be better to move that logic into https://github.com/oVirt/buildcontainer ? The plan is to use buildcontainer for api-model as a part of switching to regular RPM build